src/ui/carousel/carousel-synchronizer.ts

```diff
@@ -0,0 +1,123 @@
+import { setf } from '../../setf/setf';
+import { Carousel } from './carousel';
+
+/**
+ * Will synchronize the slide positions of multiple carousels.
+ *
+ * This can be useful if you have separate areas for assets, copy and/or
+ * navigation that all need to be synchronized together.
+ */
+export class CarouselSynchronizer {
```

Comment:
I'm a bit confused and not following how this might used in practice?  Are you provide an example?